### ISSUE TITLE
awscc_rds_db_cluster does not support aurora-postgresql engine due to forced backtrack_window

### ISSUE THREAD

**Issue by common-cranberry** [hashicorp/terraform-provider-awscc#1805]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### Affected Resource(s)

* awscc_rds_db_cluster

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
resource "awscc_rds_db_cluster" "main" {
  # ...
  engine           = "aurora-postgresql"
}

resource "awscc_rds_db_cluster" "main" {
  # ...
  backtrack_window = 0
  engine           = "aurora-postgresql"
}

resource "awscc_rds_db_cluster" "main" {
  # ...
  backtrack_window = null
  engine           = "aurora-postgresql"
}
```

### Debug Output

https://gist.github.com/common-cranberry/0e81762edaee4adce0c17c4c36fd54ad

### Expected Behavior

The provider should not send any value for `BacktrackWindow` when engine is not implicitly or explicitly `aurora-mysql` which is the only engine to support this functionality.

### Actual Behavior

The following response is tracked in CloudTrail no matter which of the above values for `backtrack_window` are provided, it is always set to `0` with this error message.
```json
{
    "eventSource": "rds.amazonaws.com",
    "eventName": "CreateDBCluster",
    "awsRegion": "us-east-1",
    "sourceIPAddress": "cloudformation.amazonaws.com",
    "userAgent": "cloudformation.amazonaws.com",
    "errorCode": "InvalidParameterValueException",
    "errorMessage": "Backtrack is not enabled for the aurora-postgresql engine.",
    "requestParameters": {
      "backtrackWindow": 0,
    }
}
```

### Steps to Reproduce

1. `terraform apply`

### References

* https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-backtrackwindow

**Comment by quixoticmonk** [comment]
Thank you for opening the issue, @common-cranberry . The reason could be the default value set in the schema used by the provider. I have opened an internal ticket to review the schema and the applicability of the property.

**Comment by quixoticmonk** [comment]
@common-cranberry  The change in the schema is available in the registry and is in 1.5.0. Can you retest ?

**Comment by quixoticmonk** [comment]
Retested the changes for the resource. Closing the issue as the backtrack_window input is updated in schema not to be set for postgres enginer.
